make/run

```diff
@@ -83,7 +83,8 @@ helm install output/helm \
      --set "env.UAA_HOST=${UAA_HOST}" \
      --set "env.UAA_PORT=${UAA_PORT}" \
      --set "env.UAA_CA_CERT=${CA_CERT}" \
-     --set "kube.external_ip=$(dig +short ${DOMAIN})" \
+     --set "kube.external_ips[0]=$(dig +short ${DOMAIN})" \
+     --set "kube.external_ips[1]=192.0.2.42" \
```

Comment:
If this is just for testing, why does it have to be checked into the repo and not just added as a comment on the Trello card?

If this is a permanent addition, then it should have a comment in the script and not just the commit message.

Reply:
This is a permanent change (to ensure that we don't regress). But I'll add comments to both places.

Reply:
Fixed in  1ccea81